src/action/services/useApi.js

```diff
@@ -0,0 +1,36 @@
+import { useState } from "react";
+import * as FetchApi from "../fetch.js";
+
+function useApi() {
+  const [isLoading, setIsLoading] = useState(false);
+
+  async function fetchData(num) {
+    try {
```

Comment:
You do not need this try and catch, because there is already a component that handles it in fetch.js - which is the requestManager

Reply:
done, deleted the try and catch, of couse requestManager doesnt exist anymore.
its inside organizeFetchingPages as requested in previous comment